pr_testing/test_multiple_prs.sh

```diff
@@ -1159,6 +1159,38 @@ for WF in ${WORKFLOWS_FOR_VALGRIND_TEST//,/ }; do
   popd
 done
 
+#
+# Enabled bot tests
+#
+for BT in ${ENABLE_BOT_TESTS}; do
+    if [ "$BT" = "PROFILING" ]; then
+        PROFILING_WORKFLOWS=$(echo $(grep "PR_TEST_MATRIX_EXTRAS_PROFILING=" $CMS_BOT_DIR/cmssw-pr-test-config | sed 's|.*=||'), | tr ' ' ','| tr ',' '\n' | grep '^[0-9]' | sort | uniq | tr '\n' ',' | sed 's|,*$||')
+         pushd $WORKSPACE
+         git clone https://github.com/cms-cmpwg/profiling.git
+         popd
+         mark_commit_status_all_prs 'profiling' 'pending' -u "${BUILD_URL}" -d "Running tests" || true
+         report_pull_request_results_all_prs_with_commit "TESTS_RUNNING" --report-pr ${REPORT_H_CODE} --pr-job-id ${BUILD_NUMBER} --add-message "Running Profling" ${NO_POST}
+         for PROFILING_WORKFLOW in $PROFILING_WORKFLOWS;do
+             $WORKSPACE/profiling/Gen_tool/Gen.sh $CMSSW_IB
+             $WORKSPACE/profiling/Gen_tool/runall.sh $CMSSW_IB
+             $WORKSPACE/profiling/Gen_tool/runall_cpu.sh $CMSSW_IB
+             $WORKSPACE/profiling/Gen_tool/runall_mem.sh $CMSSW_IB
+             pushd $WORKSPACE/$CMSSW_IB/src/$PROFILING_WORKFLOW
+             ./profile.sh $CMSSW_IB
+             popd
+             pushd $WORKSPACE/$CMSSW_IB/src
+             for f in $(find $PROFILING_WORKFLOW -type f -name '*.sql3' -o -name '*.log' -o -name '*RES*.txt' -o -name '*.json' -o -name 'sorted*.txt') ; do
+               d=$(dirname $f)
+               mkdir -p $WORKSPACE/upload/$d
+               mv $f $WORKSPACE/upload/$d/
+             done
+             popd
+         done
+         mark_commit_status_all_prs 'profiling' 'success' -u "${BUILD_URL}" -d "All OK" || true
```

Comment:
here you can add something like
```
echo 'CMSSW_PROFILING;OK,Profiling Results,See Log,profiling' >> $RESULTS_FILE
```
and then the results will be available via PR summary page